### ISSUE TITLE
errors on Windows 10

### ISSUE THREAD

**Issue by Strandedpirate** [slok/brigadeterm#27]
Doesn't work in cmder on Windows 10 :
```
λ brigadeterm --namespace brigade
character set not supported
```

Doesn't work in normal command prompt on Windows 10:
```
c:\>brigadeterm --namespace brigade
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/slok/brigadeterm/vendor/github.com/gdamore/tcell/terminfo.LookupTerminfo(0x0, 0x0, 0x0, 0x0, 0xc0423be3f0)
        /home/travis/gopath/src/github.com/slok/brigadeterm/vendor/github.com/gdamore/tcell/terminfo/terminfo.go:766 +0x10c3
github.com/slok/brigadeterm/vendor/github.com/gdamore/tcell.NewTerminfoScreen(0xc0423be360, 0x1213340, 0xc0423be360, 0xc042455e20)
        /home/travis/gopath/src/github.com/slok/brigadeterm/vendor/github.com/gdamore/tcell/tscreen.go:40 +0x67
github.com/slok/brigadeterm/vendor/github.com/gdamore/tcell.NewScreen(0xc042351f10, 0x1, 0x1, 0xc042455e70)
        /home/travis/gopath/src/github.com/slok/brigadeterm/vendor/github.com/gdamore/tcell/screen.go:203 +0x2d
github.com/slok/brigadeterm/vendor/github.com/rivo/tview.(*Application).Run(0xc042351f10, 0x0, 0x0)
        /home/travis/gopath/src/github.com/slok/brigadeterm/vendor/github.com/rivo/tview/application.go:69 +0x4b
github.com/slok/brigadeterm/pkg/ui.(*Index).Render(0xc042343640, 0xc042349f90, 0xc042351f10)
        /home/travis/gopath/src/github.com/slok/brigadeterm/pkg/ui/index.go:65 +0x72
main.(*Main).Run(0xc042004660, 0x0, 0x0)
        /home/travis/gopath/src/github.com/slok/brigadeterm/cmd/brigadeterm/main.go:69 +0x1a3
main.main()
        /home/travis/gopath/src/github.com/slok/brigadeterm/cmd/brigadeterm/main.go:96 +0x45
```

kubectl is functioning properly and context is set.
```
kubectl config current-context
stage-myclustermgmt
```

**Comment by slok** [comment]
Hi!

I'm sorry for this error :/ I don't have myself a Windows machine, but I'll try to reproduce with a VM or similar.

Are you using the latest versión? (v0.3)

**Comment by slok** [comment]
Hey @Strandedpirate! this was fixed in #44 

Closing.
